test/regression.uts

```diff
@@ -156,23 +172,73 @@ emulate_main_input(data)
 
 = Test utility functions
 
-tmpfile =  get_temp_file(autoext=".ut")
-tmpfile.startswith("/tmp/scapy")
+tmpfile = get_temp_file(autoext=".ut")
+if WINDOWS:
+    tmpfile.startswith("/tmp/scapy")
```

Comment:
I think you need a blank line after this one.

BTW, you might want to `assert()` this line and the next one, or did I miss something?